dev/.rat-excludes

```diff
@@ -99,3 +99,4 @@ spark-deps-.*
 .*tsv
 org.apache.spark.scheduler.ExternalClusterManager
 .*\.sql
+org.apache.spark.deploy.yarn.security.ServiceCredentialProvider
```

Comment:
Instead of adding these files like this, isn't it possible to just say `META-INF/services/*`?